### ISSUE TITLE
PluginValidationError: unknown hook 'pytest_fixture_plus' 

### ISSUE THREAD

**Issue by adnanmuttaleb** [smarie/python-pytest-cases#71]
I have installed `pytest_cases`, but when using it in `conftest.py`:

```
import pytest
from pytest_cases import pytest_fixture_plus, pytest_parametrize_plus

@pytest_fixture_plus(scope='module')
@pytest.mark.parametrize('name,description', [('cs', 'computer science')])
def domain(test_client, user, name, description):
  pass

```

I get the following error:

```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/_pytest/main.py", line 196, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/_pytest/main.py", line 245, in _main
INTERNALERROR>     config.hook.pytest_collection(session=session)
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/hooks.py", line 286, in __call__
INTERNALERROR>     return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/manager.py", line 93, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/manager.py", line 87, in <lambda>
INTERNALERROR>     firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/callers.py", line 208, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/callers.py", line 80, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/callers.py", line 187, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/_pytest/main.py", line 255, in pytest_collection
INTERNALERROR>     return session.perform_collect()
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/_pytest/main.py", line 455, in perform_collect
INTERNALERROR>     self.config.pluginmanager.check_pending()
INTERNALERROR>   File "/var/www/dictionary/venv/lib/python3.6/site-packages/pluggy/manager.py", line 277, in check_pending
INTERNALERROR>     % (name, hookimpl.plugin),
INTERNALERROR> pluggy.manager.PluginValidationError: unknown hook 'pytest_fixture_plus' in plugin <module 'mo3jam
.tests.conftest' from '/var/www/.../tests/conftest.py'>
```

I tried to upgrade `pluggy`, but this did not work.

**Comment by smarie** [comment]
Thanks ! I was able to reproduce this. `pytest` thinks that `pytest_fixture_plus` is a plugin because it starts with `pytest_` :)

**Comment by smarie** [comment]
A quick and dirty workaround is to rename the symbol:

```python
import pytest
from pytest_cases import pytest_fixture_plus as fixture

@fixture(scope='module')
@pytest.mark.parametrize('name,description', [('cs', 'computer science')])
def domain(test_client, user, name, description):
  pass
```

**Comment by smarie** [comment]
Fixed in 1.12.1 - sorry for the deployment delay, I actually forgot to tag :)
